### PR TITLE
*: clean up code for set SQL statement priority, let it bypass plan

### PR DESCRIPTION
**Pull request by tiancaiamao**
Priority attribute has nothing to do with plan builder, we can pass
it from SQL statement to distsql request through StmtCtx.

@winkyao @coocood

### CODE REVIEW

File: distsql/request_builder.go [hunk 1]
```diff
@@ -138,18 +139,25 @@ func (builder *RequestBuilder) getIsolationLevel(sv *variable.SessionVars) kv.Is
 	return kv.SI
 }
 
+func (builder *RequestBuilder) getKVPriority(sv *variable.SessionVars) int {
+	switch sv.StmtCtx.Priority {
+	case mysql.NoPriority, mysql.DelayedPriority:
```
Comment (winkyao) [inline distsql/request_builder.go]: mysql.DelayedPriority equal to kv.PriorityNormal?

File: executor/executor_test.go [hunk 2]
```diff
@@ -2202,7 +2202,7 @@ func (s *testContextOptionSuite) TestCoprocessorPriority(c *C) {
 	tk.MustQuery("select count(*) from t")
 	tk.MustExec("update t set id = 3")
 	tk.MustExec("delete from t")
-	tk.MustExec("insert into t values (2)")
+	tk.MustExec("insert into t select * from t limit 2")
```
Comment (winkyao) [inline executor/executor_test.go]: what is the priority of "insert into t select * from t"?

File: distsql/request_builder.go (hunk 1 above)
Comment (tiancaiamao) [reply distsql/request_builder.go]: We don't support this option, so set to default priority.

File: executor/executor_test.go (hunk 2 above)
Comment (tiancaiamao) [reply executor/executor_test.go]: Normal

**Comment by coocood** [comment]
LGTM

**Pull request merged**